examples/phi2/phi2.py

```diff
@@ -222,10 +225,15 @@ def main(raw_args=None):
 
     # only evaluate onnx generate model
     footprints = olive_run(new_json_file)  # pylint: disable=not-callable
+    output_model_path = get_output_model_path(footprints)
     if args.genai_optimization and args.inference:
-        print("GenAI optimization does not support inference")  # noqa: T201
+        # TODO(anyone): add genai generation script to examples/utils/generator.py
+        from generate import genai_run
+
+        prompts = args.prompt if isinstance(args.prompt, list) else [args.prompt]
+        for prompt in prompts:
+            genai_run(prompt, str(output_model_path.parent))
```

Comment:
do we need create the model for each prompt? or just create once and run for all prompts?

Reply:
Updated with batch inference.